packages/midscene/src/ai-model/prompt/element_inspector.ts

```diff
@@ -1,6 +1,10 @@
 import type { ResponseFormatJSONSchema } from 'openai/resources';
+import { MATCH_BY_POSITION } from '../openai';
 
 export function systemPromptToFindElement() {
+  if (MATCH_BY_POSITION) {
```

Comment:
use `getAIConfig` from 'packages/midscene/src/ai-model/openai/index.ts', so that we can override this config in browser env